src/main/java/mate/academy/spring/dto/ShoppingCartDto.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.spring.dto;
+
+import java.util.List;
+
+public class ShoppingCartDto {
```

Comment:
```suggestion
public class ShoppingCartResponseDto {
```
Let's follow the same style.

Reply:
Done